modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -162,6 +180,10 @@ export default {
 
     apos.bus.$on('context-edited', patch => {
       this.patches.push(patch);
+
+      if (!this.idleTimer) {
```

Comment:
I don't think you meant to use `!` here?

Reply:
Yeah, if there's already a timer active we don't want to reset it.

Reply:
Good point. That's not the intention.

Reply:
But it probably should be...